tests/helpers/main.ts

```diff
@@ -33,6 +36,17 @@ interface ZipNodeReturn extends ZipReturn {
   files: TestFunctionResult[]
 }
 
+let cleanupDirectories: string[] = []
```

Comment:
I know you mention this later, but I think having the comment at the top would be useful.

```suggestion
// Accumulator of temporary directories that need to be cleaned up afterwards.
let cleanupDirectories: string[] = []
```